src/libs/ComposerUtils/index.ts

```diff
@@ -23,4 +34,26 @@ function canSkipTriggerHotkeys(isSmallScreenWidth: boolean, isKeyboardShown: boo
     return (isSmallScreenWidth && DeviceCapabilities.canUseTouchScreen()) || isKeyboardShown;
 }
 
-export {getNumberOfLines, updateNumberOfLines, insertText, canSkipTriggerHotkeys};
+/**
+ * Finds the length of common suffix between two texts
+ * @param str1 - first string to compare
+ * @param str2 - next string to compare
+ * @param cursorPosition - position of cursor
+ * @returns number - Length of the common suffix
+ */
+function findCommonSuffixLength(str1: string, str2: string, cursorPosition: number) {
```

Comment:
This feels like it belongs in expensify/common str.js, no?

Reply:
At the moment it is used only with composer and requires cursor position to exactly predict which character got appended when duplicate characters are present.